websites/A/Aniorb/presence.ts

```diff
@@ -0,0 +1,182 @@
+      ).textContent;
+    presenceData.smallImageKey = "genre";
+    presenceData.buttons = [
+      {
+        label:
+          "View The Genre " +
+          document.querySelector(
+            "span.text-red-500.capitalize.px-16.font-thin.text-xl"
+          ).textContent,
+        url: document.location.href
+      }
+    ];
+  } else if (document.location.pathname.includes("/details")) {
+    presenceData.details = `Checking Synopsis`;
+    presenceData.smallImageKey = "details";
+    presenceData.state = document.querySelector(
+      "span.font-bold.text-4xl"
+    ).textContent;
+    presenceData.buttons = [
+      {
+        // details
+        label: "Check Synopsis",
+        url: document.location.href
+      },
+      {
+        // manga link
+        label: "Read Manga",
+        url: document
+          .querySelector("span.text-red-500.text-center > a")
+          .getAttribute("href")
+      }
+    ];
+  } else if (document.location.pathname.includes("/watch")) {
+    presenceData.details = document
+      .querySelector(
+        "#__next > div > div.flex.justify-between > div.w-full.justify-center.items-center.min-h-screen.lg\\:h-full.lg\\:w-10\\/12 > div > div.flex.flex-col.pb-2.xl\\:w-player.justify-between.items-center.w-full.text-white.my-4 > div.w-full.py-4.capitalize.flex.flex-col.items-start.lg\\:items-start > span"
+      )
+      .textContent.toUpperCase();
+    presenceData.state =
+      "Episode " +
+      document
+        .querySelector(
+          "#__next > div > div.flex.justify-between > div.w-full.justify-center.items-center.min-h-screen.lg\\:h-full.lg\\:w-10\\/12 > div > div.flex.flex-col.pb-2.xl\\:w-player.justify-between.items-center.w-full.text-white.my-4 > div.flex.w-full.justify-between.items-end"
+        )
+        .textContent.substr(19, 2);
+    presenceData.smallImageKey = "watching";
+    presenceData.buttons = [
+      // watch episode
+      {
+        label: "Watch Episode",
+        url: document.location.href
+      }
+    ];
+    if (iFrameVideo) {
+      timestamps = presence.getTimestamps(
+        Math.floor(currentTime),
+        Math.floor(duration)
+      );
+    } else {
+      video = document.querySelector("div > div.plyr__video-wrapper > video");
+      if (video) {
+        (currentTime = video.currentTime),
+          (duration = video.duration),
+          (paused = video.paused),
+          (timestamps = presence.getTimestamps(
+            Math.floor(currentTime),
+            Math.floor(duration)
+          ));
+      }
+    }
+
+    if (!isNaN(duration)) {
+      presenceData.smallImageKey = paused ? "pause" : "play";
+      presenceData.smallImageText = paused
+        ? (await strings).pause
+        : (await strings).play;
+      presenceData.endTimestamp = timestamps[1];
+      if (paused) {
+        delete presenceData.endTimestamp;
+      }
+    }
+  } else if (document.location.pathname.includes("/search")) {
+    presenceData.details = "Searching For...";
```

Comment:
I don't think you should include ``...`` when you are setting search text in state.